test/support/test_config.h

```diff
@@ -90,4 +90,24 @@
 #endif
 #endif //!defined(_ENABLE_RANGES_TESTING)
 
+// Enable/disable std::complex with constexpr testing
+#define TEST_COMPLEX_WITH_CONSTEXPR 0
+
+// Disable ::std::conj(int) on host
+#define TEST_COMPLEX_CONJ_FOR_NON_COMPLEX_TYPES_ON_HOST_BROKEN 1

```

Comment:
I create this define because std::conj(int) not compiled in host code at all configurations.
If this define is not applicable by any reasons, we may delete test of std::conj(int) from host code.